image-search/Models/ActivityIndicator.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ActivityIndicator.swift
+//  image-search
+//
+//  Created by Renato Vieira on 22/10/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class ActivytyIndicator: UIViewController {
+    var activityIndicator = UIActivityIndicatorView()
+        
+    override func loadView() {
+        view                 = UIView()
```

Comment:
I've never used this function of loadView, but nice approach.

Reply:
👍 :)